storage/kubernetes.go

```diff
@@ -66,3 +74,12 @@ type KubernetesFilesystemUnitAttachmentParams struct {
 	// VolumeId is the storage provider's unique identifier for the volume.
 	VolumeId string
 }
+
+// RandomPrefix returns a random string.
+func RandomPrefix() (string, error) {
```

Comment:
hmm.. so technically all 3.6.12 and below applications will have a storage unique ID generated randomly using this?